mustache.js

```diff
@@ -551,15 +547,11 @@
   };
 
   Writer.prototype.unescapedValue = function unescapedValue (token, context) {
-    var value = context.lookup(token[1]);
-    if (value != null)
-      return value;
+    return (value != null) ? value : context.lookup(token[1]);
```

Comment:
This isn't valid. You have removed the declaration of `value`, though you're still checking its contents.